README.md

```diff
@@ -0,0 +1,123 @@
+<div align="center">
+  <img src="https://raw.githubusercontent.com/apostrophecms/apostrophe/main/logo.svg" alt="ApostropheCMS logo" width="80" height="80">
+
+  <h1>Apostrophe Blog</h1>
+  <p>
+    <a aria-label="Apostrophe logo" href="https://v3.docs.apostrophecms.org">
+      <img src="https://img.shields.io/badge/MADE%20FOR%20Apostrophe%203-000000.svg?style=for-the-badge&logo=Apostrophe&labelColor=6516dd">
+    </a>
+    <a aria-label="Test status" href="https://circleci.com/gh/apostrophecms/blog/tree/main">
+      <img alt="" src="https://img.shields.io/circleci/build/github/apostrophecms/blog?labelColor=000000&logo=circleci&style=for-the-badge">
+    </a>
+    <a aria-label="Join the community on Discord" href="http://chat.apostrophecms.org">
+      <img alt="" src="https://img.shields.io/discord/517772094482677790?color=5865f2&label=Join%20the%20Discord&logo=discord&logoColor=fff&labelColor=000&style=for-the-badge&logoWidth=20">
+    </a>
+    <a aria-label="License" href="https://github.com/apostrophecms/blog/blob/main/LICENSE.md">
+      <img alt="" src="https://img.shields.io/static/v1?style=for-the-badge&labelColor=000000&label=License&message=MIT&color=3DA639">
+    </a>
+  </p>
+</div>
+
+This module bundle helps developers quickly add blog articles to Apostrophe 3 websites. It provides the blog piece type (`@apostrophecms/blog`) as well as a special page type (`@apostrophecms/blog-page`) for editors to create a blog.
```

Comment:
"blog post piece type" would help convey the difference

Reply:
Done